README.md

```diff
@@ -1,26 +1,11 @@
-<!--
-Avoid using this README file for information that is maintained or published elsewhere, e.g.:
-
-* metadata.yaml > published on Charmhub
-* documentation > published on (or linked to from) Charmhub
-* detailed contribution guide > documentation or CONTRIBUTING.md
-
-Use links instead.
--->
-
-# charmed-etcd-operator
-
-Charmhub package name: operator-template
-More information: https://charmhub.io/charmed-etcd-operator
-
-Describe your charm in one or two sentences.
+## Description
+The Charmed etcd Operator deploys and operates the [etcd](https://etcd.io) 
+software on VMs and machine clusters. The Operator in this repository is a 
```

Comment:
Why is the paragraph split over lines?

Reply:
I guessed @reneradoi 's editor auto-wraps after a certain line length. I didn't think it was a problem at first, but on second thought, it might be better to disable auto-wrapping because RTD will be in charge of that eventually.

We can re-format the files in this PR, but I am also happy to do it when I set up the RTD infrastructure in the repo in the upcoming days, since there will be some other formatting changes.

Reply:
Yeah my IDE does that when it reaches 80 chars. Will switch it off, but if we have to re-format anyway, then we can do this then.